src/components/pages/Authentication/SignUp/index.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import { connect } from 'react-redux';
+
+import Input from 'components/ui/Input';
+import withAuthenticationContainer, { AuthenticationProps } from 'components/containers/AuthenticationContainer';
+import __spacing from 'settings/__spacing';
+import { fontSizes, fontWeights } from 'settings/__fonts';
+import { BRAND_PRIMARY, BRAND_WHITE, BRAND_PRIMARY_HOVER } from 'settings/__color';
+import { signUp } from 'store/actions/auth';
+
+function SignUp(props: AuthenticationProps) {
+  const { handleChange, handleSubmit, handleBlur, values, values: { errors } } = props;
+
+  return (
+    <SignUp.Wrapper onSubmit={handleSubmit('SIGN_UP')}>
+      <Input
+        name="firstName"
+        handleChange={handleChange}
+        handleBlur={handleBlur}
+        title="First Name"
+        value={values.firstName}
+        errorFeedback={errors.firstName}
+      />
+      <Input
+        name="lastName"
+        handleChange={handleChange}
+        handleBlur={handleBlur}
+        title="Last Name"
+        value={values.lastName}
+        errorFeedback={errors.lastName}
+      />
+      <Input
+        name="email"
+        handleChange={handleChange}
+        handleBlur={handleBlur}
+        title="Email"
+        value={values.email}
+        errorFeedback={errors.email}
+      />
+      <Input
+        name="password"
+        handleChange={handleChange}
+        handleBlur={handleBlur}
+        title="Password"
+        value={values.password}
+        errorFeedback={errors.password}
+      />
+      <Input
+        name="confirmPassword"
+        handleChange={handleChange}
+        handleBlur={handleBlur}
+        title="Confirm Password"
+        value={values.confirmPassword}
+        errorFeedback={errors.confirmPassword}
+      />
+      <button type="submit">Sign Up</button>
+    </SignUp.Wrapper>
+  );
+}
+
+SignUp.Wrapper = styled.form`
```

Comment:
I think we need a way to make this form styles reuseable in the `Login` and `Reset Password` pages.